pkg/generated/restapi/configure_rekor_server.go

```diff
@@ -70,6 +71,8 @@ func configureAPI(api *operations.RekorServerAPI) http.Handler {
 	api.YamlProducer = util.YamlProducer()
 
 	api.ApplicationXPemFileProducer = runtime.TextProducer()
+	api.ApplicationTimestampQueryConsumer = runtime.TextConsumer()
```

Comment:
This may work, but wondering if `runtime.ByteStreamProducer()` and `runtime.ByteStreamConsumer()` are more appropriate